CONTRIBUTING.md

```diff
@@ -246,7 +251,11 @@ website. There are 4 ways to customize how it is displayed:
 
 ## A Note on Definitions
 
-There are many forms of Two Factor Auth, but DongleAuth.info is only interested in listing sites that support Two Factor Authentication using USB dongles. Currently that means the site must support either One Time Passwords (HOTP / RFC 4226 or TOTP / RFC 6238) or FIDO Universal 2nd Factor (U2F).
+There are many forms of Two Factor Auth, but DongleAuth.info is only interested in
+listing sites that support Two Factor Authentication using USB dongles. Currently
+that means the site must support either One Time Passwords (HOTP / RFC 4226 or TOTP / RFC 6238)
+or FIDO Universal 2nd Factor (U2F). If a site supports >1 U2F dongle, `multipleu2f`
+may be set to `Yes`.
```

Comment:
> If a site supports >1 U2F dongle, `multipleu2f` may be set to `Yes`.

You may change to "If a site supports adding multiple U2F dongles (e.g. as a backup), `multipleu2f` may be set to `Yes`."

Reply:
Sure: note that I was keeping the verbiage the same as elsewhere: this changes that.

Reply:
I know, it is just more clear for potential contributers here, I guess.